standards/python.md

```diff
@@ -31,39 +32,48 @@ See [Imports in PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and [Im
 We use [isort](https://github.com/timothycrosley/isort) to lint imports to comply with both style guidelines, and provide a [sample isort configuration](../.isort.cfg). This configuration attempts to enforce the following conventions:
 
 - Multiple-line `from` imports become grouped within parenthesis with one imported name per-line and a trailing comma for the last imported name.
-- Consider `six` to be equivalent to `__future__` imports.
 - Django imports are included as their own section between the standard library and third-party imports.
 - For Wagtail-specific projects like [cfgov-refresh](https://github.com/cfpb/cfgov-refresh), Wagtail imports are included as their own section between Django and third-party imports.
 
 The isort configuration can go into a `tox.ini` file under an `[isort]` header.
 
-## Python version
+## Dependency support/testing matrix
 
-We test against Python 3.6.
+We have two general types of Python projects:
 
-We recommend using [tox](https://tox.readthedocs.io/en/latest/) for matrix testing, and provide a [sample tox configuration](../tox.ini) that tests against Python 3.6 and Django LTS versions 1.8 and 1.11.
+- Applications that have a running instance, e.g., [cfgov-refresh](https://github.com/cfpb/cfgov-refresh)
+- Libraries that provide some functionality independent of applications
 
-For writing code that is Python 3-compatible, please see the [Python Porting HOWTO](https://docs.python.org/3/howto/pyporting.html) and the [Django Porting to Python3](https://docs.djangoproject.com/en/1.11/topics/python3/). In particular, we prefer to:
+We use a the following rubric to determine which versions of Python and major dependencies should be supported. Assuming the project has Django and Wagtail as its major dependencies, our recommendation would be this:
```

Comment:
Minor corrections to the text I suggested earlier (removing a typo, and minor wording change for style)

```suggestion
We use a rubric like the one below to determine which versions of Python and major dependencies should be supported. Assuming the project has Django and Wagtail as its major dependencies, our recommendation would be this:
```